src/spikeinterface/sorters/internal/tridesclous2.py

```diff
@@ -52,6 +51,20 @@ class Tridesclous2Sorter(ComponentsBasedSorter):
         "save_array": True,
     }
 
+    _params_description = {
+        "apply_preprocessing": "Apply internal preprocessing or not",
+        "waveforms": "A dictonary containing waveforms params: ms_before, ms_after, radius_um",
+        "filtering": "A dictonary containing filtering params: freq_min, freq_max", 
+        "detection": "A dictonary containing detection params: peak_sign, detect_threshold, exclude_sweep_ms, radius_um", 
+        "selection": "A dictonary containing selection params: n_peaks_per_channel, min_n_peaks", 
+        "svd": "A dictonary containing svd params: n_components", 
+        "clustering": "A dictonary containing clustering params: split_radius_um, merge_radius_um",
+        "templates": "A dictonary containing waveforms params for peeler: ms_before, ms_after",
+        "matching": "A dictonary containing matching params for matching: peak_shift_ms, radius_um",
+        "job_kwargs": "A dictionnary containing job kwargs",
```

Comment:
```suggestion
        "job_kwargs": "A dictionary containing job kwargs",
```

Thanks Sam for adding the descriptions :)